governance/01-governance.md

```diff
@@ -8,33 +8,78 @@ All repositories in the Kyma organization must follow the official guidelines, c
+* Approve only those PRs in which the contributor made the requested improvements
+* Check if the related CI tests have completed successfully before approving the PR
+* Make sure that the PR approval flow runs smoothly
+* Proactively fix bugs
+* Perform maintenance tasks for their projects
+* An issue that is assigned to a current release is treated with priority by the maintainer that is the assignee
+* Monitor [Slack](http://slack.kyma-project.io)
+* Triage GitHub issues and perform pull request reviews for other maintainers and the community. Follow the [labels workflow](/governance/#issues-workflow-issues-workflow) document
+* In general, continue to be willing to spend at least 50% of one's time working on Kyma (~2.5 business days per week).
+
+### Process for becoming a maintainer (aka codeowner)
+
+* Express interest to the existing maintainers that you or your organization are interested in becoming a
+  maintainer. Depending on whether you are an individual or a corporate contributor/maintainer, different contributor license agreement applies.
+* Express what the areas of Kyma that you want to engage with are. Kyma, because of its goal, is complex, and it is impossible to become a maintainer in every area. Make sure you get support from a proper group of maintainers.
+* Becoming a maintainer generally means that you are going to be spending a substantial part of your time (>50%) on Kyma for the foreseeable future. Among other responsibilities, this means:
+  * creating and merging pull requests
+  * reviewing pull requests
+  * creating or commenting on issues
+  * helping on Slack and other community communication channels.
+* We will expect you to start contributing increasingly complicated PRs, under the guidance
+  of the existing maintainers.
+* We may ask you to do some PRs from our backlog.
+* As you gain experience with the code base and our standards, we will ask you to do code reviews for incoming PRs (i.e., all maintainers are expected to shoulder a proportional share of community reviews).
+* After a period of approximately 3 months of working together and making sure we see eye to eye, a candidate for maintainer should create an issue in the [community](https://github.com/kyma-project/community) repository with a list of his/her current contributions. The existing maintainers will confer and decide whether to grant the maintainer status or not. While we make no guarantees on the length of time this will take, we expect it to be no longer than 3 months.
+
+>**NOTE:** To be accepted as a maintainer and be added to Kyma GitHub organizations, you need to have [two-factor authentication](https://help.github.com/en/articles/about-two-factor-authentication) enabled on your GitHub account.
+
+### When does a maintainer lose the maintainer status?
+
+- If a maintainer is no longer interested or cannot perform the maintainer duties listed above, they
```

Comment:
```suggestion
- If a maintainer is no longer interested or cannot perform the maintainer duties listed above, he/she
```

Sorry, I missed it before, but since we used `he/she` in other places, let's be consistent here as well.